src/main/java/com/scottlogic/practice/mealscheduler/Models/Meal.java

```diff
@@ -1,25 +1,39 @@
 package com.scottlogic.practice.mealscheduler.Models;
 
-import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
 import java.time.LocalDateTime;
-import java.util.UUID;
 
-@Data
+/*not using @Data class as IntelliJ says it is bad to use it with @Entity*/
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
 public class Meal {
-    private UUID Id;
+    public static Meal copyOf(Meal meal) {
+        return new Meal(meal.getId(), meal.getName(), meal.getTime(), meal.getUser());
+    }
+
+    @Id
+    @GeneratedValue
+    @Getter
+    @Setter
+    private int Id;
+
+    @Getter
+    @Setter
     private String Name;
-    private LocalDateTime Time;
-    private String User;
 
-    public Meal(UUID id, String name, LocalDateTime time, String user) {
-        Id = id;
-        Name = name;
-        Time = time;
-        User = user;
-    }
+    @Getter
+    @Setter
+    private LocalDateTime Time;
 
-    public Meal(Meal meal) {
-        this(meal.Id, meal.Name, meal.Time, meal.User);
-    }
+    @Getter
+    @Setter
```

Comment:
If every field has a Getter and Setter, I believe we can add the annotations at class level rather than each field, thoughts?

Reply:
Good to know, let me fix, did not realise it was possible. Thanks, good spot.